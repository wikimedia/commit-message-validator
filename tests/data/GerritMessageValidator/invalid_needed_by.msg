This needed-by is not valid

Needed-By: Ideafbeef
Change-Id: If89d24838e326fe25fe867d02181eebcfbb0e196
