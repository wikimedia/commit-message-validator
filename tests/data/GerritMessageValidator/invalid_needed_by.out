The following errors were found:
- Line 3: Needed-By: value must be a single Gerrit change id
